docs/fields/field-details.asciidoc

```diff
@@ -8065,15 +8065,59 @@ example: `2016-05-23T08:05:34.853Z`
 
 a| Unique identifier for the process.
 
-The implementation of this is specified by the data source, but some examples of what could be used here are a process-generated UUID, Sysmon Process GUIDs, or a hash of some uniquely identifying components of a process.
-
 Constructing a globally unique identifier is a common practice to mitigate PID reuse as well as to identify a specific process over time, across multiple monitored hosts.
 
+entity_id SHOULD be generated with the below methods. These methods will generate a unique and reproducible entity_id, so that if the same process is observed by multiple collectors, it can be identified as the same process.
+
+Windows:
+
+A String formatted as: "MachineGUID__PID__StartTimeSeconds__StartTimeMilliseconds"
+
+MachineGUID:: The Windows Machine GUID, which can be read from the Windows registry
```

Comment:
I wonder if we would ever had a need to know that the entity was generated by an Endpoint receiving it.